config/opensearch_dashboards.yml

```diff
@@ -281,6 +281,14 @@
 #    'ff00::/8',
 # ]
 
+# Optional setting that enables you to specify a path to PEM files for the certificate
+# authority for your connected datasources.
+#data_source.ssl.certificateAuthorities: [ "/path/to/your/CA.pem" ]
+
+# To disregard the validity of SSL certificates for connected data sources, change this setting's value to 'none'.
+# Possible values include full, certificate and none
+#data_source.ssl.verificationMode: full
```

Comment:
Are these explained in a README doc?

Reply:
@AMoo-Miki Where do you think would be best to document this? These settings are already defined for the default opensearch via the settings [`opensearch.ssl.verificationMode` and `opensearch.ssl.certificationAuthorities`](https://opensearch.org/docs/latest/install-and-configure/install-dashboards/tls/) and this PR extends those to datasources. These settings would be commonly used for all configured datasources, although I filed another [issue](https://github.com/opensearch-project/OpenSearch-Dashboards/issues/6226) on per datasource configuration.

Here are the [analogous entries](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/config/opensearch_dashboards.yml#L84-L89) for default OpenSearch.

Reply:
Adding some more info to `src/plugins/data_source/README` would be great. As a user, I find OSD's documentation very inadequate and this small step would help users find answers sooner.